clubs/README.md

```diff
@@ -174,35 +190,194 @@ keep building their website to make it unique. At the second meeting, you'll
-of the meeting will be free-form time where hackers will continue expanding on
-their portfolio websites. At the end of the meeting, everyone will come up to
-the front of the space and demo their website. As with the first meeting,
-bringing food is a good idea :wink:.
+The second meeting will pick up where the first left off. The majority of the
+meeting will be free-form time for hackers to continue expanding their portfolio
+websites. At the end of the meeting, everyone will come up to the front of the
+room and demo their website. Like the first meeting, bringing food is a good
+idea :wink:.
 
 Here's our suggested schedule:
 
 1. Wait for hackers to arrive (5 minutes)
 2. Announcements (5 minutes)
+3. Example demo (5 minutes)
 3. Free-form hacking (45 minutes)
-4. Demos (20 minutes)
+4. Demos (30 minutes)
+
+#### Wait for Hackers to Arrive
+
+This'll be very similar to the first meeting. Put on a playlist (not too loud)
+and have a slide on the projector to let people know they're in the right place.
+
+#### Announcements
+
+Announcements are your opportunity to share any updates with your members and
+let them know what they'll be doing in this meeting. In addition to sharing the
+meeting's schedule, they're useful for sharing upcoming events, upcoming guest
+visits, and any other information relevant to your members.
+
+Tell your members that for this meeting, they're going to continue where they
+left off and continue building their websites. 30 minutes before the end of the
+meeting, everyone's going to come up to the front and demo the websites they
+built. Right after announcements you're going to give an example of what a demo
+looks like.
+
+#### Example Demo
+
+You'll want to give an example of what a demo looks like so your members know
+what to expect at the end of the meeting. The best way to do this is to demo a
+project that you or another person built for the Portfolio workshop.
+
+If you don't have a website that you built to demo, go ahead and pull up one of
+the Portfolio workshop examples that were shown in the first meeting and use it.
+[Click here][demo] for an example of what a project demo looks like (this demo
+was given at a conference).
+
+[demo]: https://www.youtube.com/watch?v=MY01d647S9Y
+
+#### Free-Form Hacking
+
+This'll basically be the same as the free-form hacking section in the first
+meeting. Members will continue building their websites and your role is to
+create an environment where people are comfortable for asking for help.
+
+Just like in the first meeting, when people do have questions, be eager to help
+them out. It's also a good idea to put on an upbeat playlist (but again, not too
+loud -- people should feel comfortable talking with one another).
+
+#### Demos
+
+Before getting to the demos themselves, give your members 45, 30, 15, and 5
+minute warnings so the demos aren't a surprise.
+
+30 minutes before demos start, have everyone email you a link to their website.
```

Comment:
Does the first workshop onboard members into Slack? If it does, then we can
use Slack (which would be neat-o).

On Mon, Jan 18, 2016 at 12:14 PM, Jonathan Leung <notifications@github.com>
wrote:

> In clubs/README.md
> <https://github.com/hackclub/hackclub/pull/657#discussion_r50039767>:
>
> > +#### Free-Form Hacking
> > +
> > +This'll basically be the same as the free-form hacking section in the first
> > +meeting. Members will continue building their websites and your role is to
> > +create an environment where people are comfortable for asking for help.
> > +
> > +Just like in the first meeting, when people do have questions, be eager to help
> > +them out. It's also a good idea to put on an upbeat playlist (but again, not too
> > +loud -- people should feel comfortable talking with one another).
> > +
> > +#### Demos
> > +
> > +Before getting to the demos themselves, give your members 45, 30, 15, and 5
> > +minute warnings so the demos aren't a surprise.
> > +
> > +30 minutes before demos start, have everyone email you a link to their website.
>
> Email is terrible... What if we made a Google form where people submitted
> their demos and that would mean we would have their demo links as well?
> Navigating a spreadsheet is also A LOT easier than navigating an email
> account when trying to pull up live demo links.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/657/files#r50039767>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>